lib/Doctrine/deploy/sampleData/ServiceEndpoints.xml

```diff
@@ -285,11 +285,385 @@
+        <DESCRIPTION>Debrecen Tech's local LFC</DESCRIPTION>
+        <SERVICE_TYPE>Local-LFC</SERVICE_TYPE>
+        <HOST_IP>197.344.80.70</HOST_IP>
+        <SCOPE>Local</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Debrecen Tech</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="11069G0">
+        <PRIMARY_KEY>11069G0</PRIMARY_KEY>
+        <HOSTNAME>bdii.debrecentech.hu</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=11069&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>Debrecen Tech's Site BDII</DESCRIPTION>
+        <SERVICE_TYPE>Site-BDII</SERVICE_TYPE>
+        <HOST_IP>197.800.13.14</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>N</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Debrecen Tech</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="5711G0">
+        <PRIMARY_KEY>5711G0</PRIMARY_KEY>
+        <HOSTNAME>bdii.barcs-drava-stf.hu</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=5711&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>Site BDII</DESCRIPTION>
+        <SERVICE_TYPE>Site-BDII</SERVICE_TYPE>
+        <HOST_IP>190.10.390.27</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Barcs-Drava STF</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="19042G0">
+        <PRIMARY_KEY>19042G0</PRIMARY_KEY>
+        <HOSTNAME>SRM.barcs-drava-stf.hu</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=19042&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>barcs-drava-stf SRM</DESCRIPTION>
+        <SERVICE_TYPE>SRM</SERVICE_TYPE>
+        <HOST_IP>193.340.05.02</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Barcs-Drava STF</SITENAME>
+        <COUNTRY_NAME>Hungary</COUNTRY_NAME>
+        <COUNTRY_CODE>HU</COUNTRY_CODE>
+        <ROC_NAME>NGI_HU</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="9293G0">
+        <PRIMARY_KEY>9293G0</PRIMARY_KEY>
+        <HOSTNAME>site-bdii.danubeinstitute.hu</HOSTNAME>
```

Comment:
perhaps change 'institute' to some synonym - here and elsewhere.

Reply:
ok